components/Article/TableOfContents/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { FormattedMessage } from 'react-intl';
+import Link from 'next/link';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+
+type PropsTableOfContents = {
+  depth: number;
+  value: string;
+  id: string;
+}[];
+
+const traverseTableOfContents = (tableOfContents: PropsTableOfContents) => {
+  return (
+    <ul>
+      {tableOfContents.map(item => {
+        return (
+          <li key={item.id}>
+            <Link href={`#${item.id}`}>{item.value}</Link>
+          </li>
+        );
+      })}
+    </ul>
+  );
+};
+const TableOfContents: FC<{ tableOfContents: PropsTableOfContents }> = ({
+  tableOfContents,
+}) => {
+  if (!tableOfContents.length) {
```

Comment:
```suggestion
  if (tableOfContents.length === 0) {
```
It's better to have positive checks than negations.